rest/models/model_rows_update.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+// RowsUpdate defines an update operation on rows within a table.
+type RowsUpdate struct {
+	Changeset []Changeset `json:"changeset" validate:"required"`
```

Comment:
Should/Could this use `models.Column`? It's weird that this is a different type then in row add.

Reply:
Really it was just a matter of semantics. I used Changeset here because it's an update. Also they were initially different structs but have mostly converged now.

Reply:
Given that in C* everything is an upsert, I think it makes sense to have the same structure for both.